worlds/overcooked2/Options.py

```diff
@@ -9,6 +9,12 @@ class LocationBalancingMode(Enum):
     full = 2
 
 
+class DeathLinkMode(Enum):
```

Comment:
You seem to explicitly assign int values for this Enum. Maybe this should be https://docs.python.org/3/library/enum.html#enum.IntEnum ?

Reply:
This was driving me *insane* about Python for like the last 3 years. Thank you so much for this gift.